extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -534,7 +534,8 @@ export class ImageViewer {
 
     const newPosX = this.boundX_(this.startX_ + deltaX * newScale, false);
     const newPosY = this.boundY_(this.startY_ + deltaY * newScale, false);
-    return this.set_(newScale, newPosX, newPosY, animate);
+    return /** @type {!Promise} */ (
```

Comment:
Likewise, `!Promise|undefined`. Also I don't think you need type conversion here.